scalableminds/webknossos
# Show wk user in vx workflow list

### Steps to test:
- Install voxelytics and follow instructions in worker readme (set directory and enable jobs)
- start a job such as computing a mesh file
- go to voxelytics overview and see your current wk user name in table
- try out filter of user column 

### TODOs:
- [x] host user only as backup for wk user
- [x] view workflows of multiple users -> change in pgadmin
- [x] implement user filter

### Issues:
- fixes #7735

------
(Please delete unneeded items, merge only when none are left open)
- [x] :exclamation:  **Removed dev-only changes like prints and application.conf edits** :exclamation:
